### PR TITLE
Add addmission count down

### PR DESCRIPTION
**Pull request by erlingfn**
Fix: https://github.com/webkom/committee-admissions/issues/221
Styling feedback is highly appreciated
image
image

### CODE REVIEW

**Comment by cherrybeans** [comment]
Nice! Prøv å fiks stylingen slik som i bildet i issuen. Rød, bold text, i en viss avstand til knappen :)

File: frontend/src/routes/LandingPage/index.js [hunk 1]
```diff
@@ -140,6 +141,9 @@ class LandingPage extends Component {
             </li>
           )}
         </LinkWrapper>
+        {admission.is_open && (
```
Comment (JonasBak) [inline frontend/src/routes/LandingPage/index.js]: Shouldn't this be when the admission is _not_ open

File: frontend/src/components/AdmissionCountDown/index.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      1000
+    );
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.counterInterval);
+  }
+
+  calcTimeDiff() {
+    const now = moment();
```
Comment (mrfarstad) [inline frontend/src/components/AdmissionCountDown/index.js]: Move this constant outside the function so you don't have to declare another similar constant later :)

File: frontend/src/routes/LandingPage/index.js (hunk 1 above)
Comment (erlingfn) [reply frontend/src/routes/LandingPage/index.js]: Yes, thats true

**Comment by mrfarstad** [comment]
LGTM 💯

**Pull request merged**